### ISSUE TITLE
Emudeck-Installation not asking about Rom-Placement

### ISSUE THREAD

**Issue by dotbanana** [dragoonDorise/EmuDeck#256]
Hey there, 

i today installed Emudeck. I saw in Tutorial-Vids that it asks you if you want to put Roms on SD-Card or Internal.  
I never got this.
I only got the window which let me choose between Internal and Custom with round checkboxes in Expert Mode, In Easy Mode it just immediatly installs everything on NVMe.   
The SD-Card is normally integrated into Steam. I can install Games there.
Since i would like to place my Roms on SD-Card i have no idea what directory to put into this custom-field now.  
Going into root filesystem of Steamdeck searching for it doesn't help because everything is kind of chaotic and 
symlinked. I tried couple of folders out of guess but always got a permission error. Do i need to start the program as root then?
How do i do that? 
Might the problem be that i have in rootfs /run/media/ two folders? mmcblk0p1 and mmcblk0p2  
The first one is empty the second has another linux-filesystem in it, it seems.

**Comment by Godsbane** [comment]
I see you closed this - but what was your resolution?

**Comment by dotbanana** [comment]
Hey there. 
I reformatted my SD Card with a tool called 'SD Card Formatter' by SD Association.  
I gave the Card the name "SD" in Windows and formatted it again in Steam to use with the Deck. 
When looking in /run/media there are now mmcblk0p1 and SD. 
It seems the format-operation by Steam on the SteamDeck is not that good and there is stuff remaining. 
Now the install worked and i could both in Easy Mode and also in Expert Mode choose the SD Card as target.
